### DISCUSSION TITLE
2005/04/26/%E8%88%AA%E7%A9%BA%E6%AF%8D%E8%88%B0

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#14] [Announcements]
# 2005/04/26/%E8%88%AA%E7%A9%BA%E6%AF%8D%E8%88%B0

https://blog.qizhen.xyz/2005/04/26/%E8%88%AA%E7%A9%BA%E6%AF%8D%E8%88%B0

**Comment by ruanqizhen** [comment]
之前博客下的留言：
```
Unknown
2005年04月26日 12:12 
这个黑人是谁呀？
```
